### DISCUSSION TITLE
Template Update

### DISCUSSION THREAD

**Discussion by Bronya-Rand** [imgradeone/revolution#4] [General]
Hi Doki Revolution Team.

I know what I may be saying is in english which may be a barrier in this discussion but I highly recommend if you haven't already to update the template that is currently being used as the chinese template over to the 2.0 template as many of the modders on the r/DDLCMods subreddit, discord and more are using it more and more for DDLC base features that are missing from the MAS developers template and has full Ren'Py 7 and Android support. I don't know if this will be recieved by you guys but this is my part to let you guys know about it on both Revolution and old Revolution site so that chinese modders can work on a full template copy of the game easily than of the MAS team's strange edits. If you feel that you need permission to translate it, I don't mind it as long as I still get credit in the 2.0 chinese translation update.

**Comment by Bronya-Rand** [comment]
In case that you are wondering what is being asked here, it is mainly to update the chinese template to 2.0 with chinese support and defunct the older MAS template i.e remove `DDLC Mod Template（原版模板)` and keep just `DDLC Mod Template（v2 英文模板)` only.

**Comment by imgradeone** [comment]
First of all, sorry for my late.

Then...

OK, as for the Chinese Mod Template, we've patched to get the latest updates, which is the same as v2.4.6 from upstream.

For the old website (https://dokimod.cn), we have no plan for any content updates, as we've moved focus on the new Revolution site (https://revolution.dokimod.cn), and the old site did really poor. Also, we've added a notice to encourage visitors to view the new site.

As for the old template, **we've removed the link of the old template** from [this page](https://revolution.dokimod.cn/related/). The legacy version of DDLC Chinese Mod Template will have slower maintenance.

**Reply by Bronya-Rand** [reply]
No worries. Just letting y'all know about it since people may run into issues in the future with the MAS template or the old chinese mod template y'all use.

Also I will like to point out that on the Revolution site when I looked into the links to find the template, I see you guys link DokiMods. That site is currently banned on like r/DDLC and the mods subreddit since it really has had a notorious past within the community despite their staff changes and renames (like pirating plugins and then later getting people malware or weird ads on their phones and PCs. Users on the discord complained about it and later got shut down for it.) 

The sub is mainly strict due to them trying to respect the wishes of authors (they say you can't use this art at all, you can't use it at all) and they allow people to fix these mods for these issues. Really the only strict closed on is mods that are removed cause of authors being danger to minors in the community. Just letting y'all know on this.

Anyways, apart from that I can see y'all working on the translations to it. Hope this saves some headaches for y'all.

**Reply by imgradeone** [reply]
Okay.

As you mentioned, now I will take down DokiMods from [DDLC-Related Websites Page](https://revolution.dokimod.cn/related/) as we (or actually, me?) should care for creators.

> Also DokiMod (aka dokimod.cn) is **NOT AFFILIATED** with DokiMods (only that DokiMods has a letter "s").

Thank you, then.

By the way, I would be appreciated if DokiMod could have some corporation (maybe this word?) with the Global DDLC Community. As you may saw, DokiMod has a [Mod Library (or repository?)](https://revolution.dokimod.cn/modlib/) which will contain some great DDLC Mod works. This can be from Chinese developers / translators and the Global DDLC Community creators.

**Reply by Bronya-Rand** [reply]
> Okay.
> 
> As you mentioned, now I will take down DokiMods from [DDLC-Related Websites Page](https://revolution.dokimod.cn/related/) as we (or actually, me?) should care for creators.
> 
> > Also DokiMod (aka dokimod.cn) is **NOT AFFILIATED** with DokiMods (only that DokiMods has a letter "s").
> 
> Thank you, then.
> 
> By the way, I would be appreciated if DokiMod could have some corporation (maybe this word?) with the Global DDLC Community. As you may saw, DokiMod has a [Mod Library (or repository?)](https://revolution.dokimod.cn/modlib/) which will contain some great DDLC Mod works. This can be from Chinese developers / translators and the Global DDLC Community creators.

I can possibly do that with the team. However DDLC.top contains projects that have been recently banned due to authors being a danger to minors in the community like Just Yuri and possibly a few more. The team will probably not like that DDLC.top which is linked in the Doki Revolution site contains such projects (even if they are just translated). (Also maybe probably remove the DokiMods mention in the description along with DDLCMods).

Additionally I do see that under the Reddit r/DDLCMods button is a description that says `比 DokiMods 团队严格的 Mod 收录区` which translated means "Mod collection area stricter than the DokiMods team". Not sure if you forgot to change that but just a FYI about it.

**Reply by imgradeone** [reply]
> Additionally I do see that under the Reddit r/DDLCMods button is a description that says 比 DokiMods 团队严格的 Mod 收录区 which translated means "Mod collection area stricter than the DokiMods team". Not sure if you forgot to change that but just a FYI about it.

Damn, I forgot it. I will edit it later.

For the DDLC.top, I will change the description, but still keep the link, as the Chinese DDLC Community is so poor.

*edited: and now I've edited it.
